stdlib/public/core/ExistentialCollection.swift.gyb

```diff
@@ -132,6 +132,28 @@ internal class _AnySequenceBox<Element> {
 
   internal func _underestimateCount() -> Int { _abstract() }
 
+  internal func _map<T>(@noescape transform: Element throws -> T) rethrows -> [T] { _abstract(); }
+  
+  internal func _filter(@noescape includeElement: Element throws -> Bool
+  ) rethrows -> [Element] {
+     _abstract();
+  }
+  internal func _forEach(@noescape body: Element throws -> Void) rethrows { _abstract(); }
+  
+  internal func _dropFirst(n: Int) -> AnySequence<Element> { _abstract(); }
+  
+  internal func _dropLast(n: Int) -> AnySequence<Element> { _abstract(); }
+  
+  internal func _prefix(maxLength: Int) -> AnySequence<Element> { _abstract(); }
+  
+  internal func _suffix(maxLength: Int) -> AnySequence<Element> { _abstract(); }
+  
+  internal func _split(maxSplit: Int, allowEmptySlices: Bool, 
+    @noescape isSeparator: Element throws -> Bool
+  ) rethrows -> [AnySequence<Element>] {
```

Comment:
Thanks, I'll fix this. FYI I copied this formatting from split in [Sequence.swift](https://github.com/apple/swift/blob/master/stdlib/public/core/Sequence.swift#L180)